pkg/util/pki/csr_test.go

```diff
@@ -553,3 +559,140 @@ func Test_buildKeyUsagesExtensionsForCertificate(t *testing.T) {
 		})
 	}
 }
+
+func TestSignCSRTemplate(t *testing.T) {
+	// We want to be testing the behavior of SignCSRTemplate in various
+	// contexts; for that, we construct a chain of three certificates: a root
+	// CA, an intermediate CA, and a leaf certificate.
```

Comment:
This comment needs updating to reflect the new chain of certs

Reply:
Updated, thanks! Feel free to suggest re-wording. English is not my native language.